content/about.html

```diff
@@ -77,14 +77,16 @@ <h1>Jeffrey Maxwell, BBA, Treasurer</h1>
 		<p>He worked at a lifestyle retreat for nearly two years in the north Georgia mountains and has served in other ministry lines, as well. His passion is for health and self-development.</p>
 	</article>
 	<article>
-		<h1>Sarah Brandauer, Secretary</h1>
-		<img src="/img/staff/2018/2018SarahB.jpg"/>
-		<p>Sarah works in her family's media ministry "To All The World" helping with the business end as well as filming. She is also an EMT and shepherdess. Her main joys are in helping others find Christ, working with children, singing, raising wool sheep, gardening and cooking.</p>
+		<h1>Sarah Privat, Secretary</h1>
+		<!-- <img src="/img/staff/2018/2018SarahB.jpg"/> -->
+		<p>Bio to come!</p>
+		<!-- <p>Sarah works in her family's media ministry "To All The World" helping with the business end as well as filming. She is also an EMT and shepherdess. Her main joys are in helping others find Christ, working with children, singing, raising wool sheep, gardening and cooking.</p> -->
```

Comment:
I like to delete the old paragraph content in one shot. That way it's not confusing - especially in this case where we are going from "Sarah" to "Sarah"! :P

Reply:
Okay, done!